### ISSUE TITLE
Keyboard not working

### ISSUE THREAD

**Issue by Djago** [RainbowMiner/RainbowMiner#1444]
Hi

Recently something has happened and now I can use the keyboard anymore on the console. I press any of the bracketed keys, but it does nothing. A couple of days (weeks?) ago, it worked ok.

Regards,

**Comment by RainbowMiner** [comment]
Hi, that's weird! Actually, nothing has been changed, regarding the Console. Do try the following:
- hit Windows key, then type `pwsh` and start a PowerShell Console
- now enter `Write-Host "Hello World!"` and hit enter.

Did this work?

**Comment by Djago** [comment]
It worked. But the rbminer console does not read my keyboard!

**Comment by RainbowMiner** [comment]
Could you please create a Debug file and upload it? Maybe, we can find a hint in the logs.
Also, did you try to restart RainbowMiner? In your case, it would mean, close the Console by clicking on the Windows "X" and restart.

**Comment by Djago** [comment]
Uploding...
BTW; I have Powershell 7.1 and RBMiner recommends 7.1.2. Should I try this first or you want me to try like it is now?
BTW2: I'm an IT guy and I'm with RBMiner for a long time, so feel free to ask me whatever you want and I'll try to help you to the extent of my knowledge.
[debug_2021-02-24.zip](https://github.com/RainbowMiner/RainbowMiner/files/6036495/debug_2021-02-24.zip)
BTW3: I've asked a question in the old #757 thread, but it's already answered. I don't know if it's ok for you to ask there or if you prefer me to start a new question

**Comment by RainbowMiner** [comment]
Yes, try to upgrade the Powershell Core to v7.1.2 - that eliminates the possibility of a bug in the v7.1

**Comment by Djago** [comment]
I'm on 7.1.2. No joy here. Still it doesn't recognize my key presses. I've also noted that now I can't click with the mouse in the console. Before this actual problem, if I mistakenly click over the console, a cursor block appeared and the console waited there for my enter key. That was a problem because sometimes when I inadvertently clicked on the console, the script just stopped waiting for me.
Now if I click over the console, nothing happens. I can RIGHT click and select an action from the menu (like selecting, copying and the like), but single left click won't do anything. I think this is related to the key press problem.

**Comment by RainbowMiner** [comment]
That is really strange. Also, the uploaded Debug file seems to be empty. Could you please try to create another one?

**Comment by Djago** [comment]
[debug_2021-02-24.zip](https://github.com/RainbowMiner/RainbowMiner/files/6037236/debug_2021-02-24.zip)

**Comment by RainbowMiner** [comment]
Ok, I have got an idea. Did you by any chance install the Powershell for the current user, only?
Try:
- hit Windows key and type "pwsh" - **now start pwsh as administrator**
- try to type in it. Does it work?

**Comment by Djago** [comment]
I have a shortcut to start.bat on my desktop and it has the "Run as Administrator" checked.
But I've tried as you said. Open a pwsh manually (as Administrator) and then, from that console, execute the start.bat.
No joy either :'(

**Comment by RainbowMiner** [comment]
I am pretty confident, that it must have something to do with user rights on Windows. Is your current user of type "Administrator" or other?

**Comment by Djago** [comment]
I have only this user on my computer and it is an Administrator.
The strange part is that, it WAS working ok a couple of days (week?) before and I'm certain because I've changed some configuration here and there...

**Comment by RainbowMiner** [comment]
Did you manage to regain control to the Console?

**Comment by Djago** [comment]
No, I can't control the console anymore... BUT today I've discovered something strange (at least for me).
If I put RBMiner on pause via the webfig, when RBMIner is paused I CAN send key strokes.

**Comment by RainbowMiner** [comment]
That's like superweird! I will find out.

**Comment by Djago** [comment]
I've found something. I don't know what it means and if it's related.
The line containing the Profit is different from the rest. It's a white background with black text, and there are some blocks there too:
![image](https://user-images.githubusercontent.com/34612740/111235477-7bff8580-85cf-11eb-89b9-b6757a9ec8e4.png)
Unfortunately it is there too when in pause (and when in pause, the keyboard works OK).
I remember that when the keyboard was working and I clicked over the console, the zone I clicked over displayed a block (like a cursor), and rbminer stayed there until I pressed enter.

**Comment by aliyazdi75** [comment]
@RainbowMiner I also face this issue. The keyboard shortcut doesn't work. And I can only manage that by localhost.

**Comment by IdiocracyFR** [comment]
Same issue since updating powershell to newest version. Going to try downgrading.

**Comment by IdiocracyFR** [comment]
Working with powershell 7.2.7.0
might also work with other newer versions. Does not work with newest version of rb miner & newest version of powershell together, at least on my win 10 system.
